index.php

```diff
@@ -1,181 +1,13 @@
 <?php
+use Twig\Environment;
+use Twig\Loader\FilesystemLoader;
+
+include_once 'vendor/autoload.php';
 include_once 'language.php';
 include_once 'api/get_rbg_streams.php';
 
-function td($el)
-{
-    return "<td>$el</td>";
-}
-
-function tr($el)
-{
-    return "<tr>$el</tr>";
-}
-
-
-function constructLink($type, $link) {
-    if ($type == "comb") {
-        return '<a href="./watch/' . $link . '"><span class="material-icons md-dashboard"></span></a>';
-    } else if ($type == "pres") {
-        return '<a href="./watch/' . $link . '"><span class="material-icons md-present_to_all"></span></a>';
-    } else if ($type == "cam") {
-        return '<a href="./watch/' . $link . '"><span class="material-icons md-videocam"></span></a>';
-    }
-}
```

Comment:
I think now it makes even more sense, why it was a good idea not to have business logic in this function - good that we cleaned this up in the previous PR already :+1: :1st_place_medal: